src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/_upgrade_extension.py

```diff
@@ -13,18 +13,19 @@
 
 @register_command(
     "connectedmachine upgrade-extension",
+    is_preview=True,
```

Comment:
May I ask why do you mark the GA command as `is_preview`? Please note that even if using the preview api-version for this command, if its interaction UX is stable for CLI customers, there is no need to mark it as `is_preview`, otherwise it may be a regression of stability for customers.

Reply:
I agree. Fixed to be stable instead.